portfolio/src/main/webapp/index.html

```diff
@@ -1,19 +1,95 @@
 <!DOCTYPE html>
 <html>
   <head>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <link rel="icon" type="image/png" href="./resources/terminal.png"/>
+    <title>Valentina Brambila Grando</title>
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.2/css/all.css" integrity="sha384-vSIIfh2YWi9wW0r9iZe7RJPrKwp6bG+s9QZMoITbCckVJqGCCRhc+ccxNcdpHuYu" crossorigin="anonymous">
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
     <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
+    
   </head>
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click <a href="/hello">here</a> to see content created by server code.</p>
-      <p>Click here to get a random message:</p>
+    <div class="jumbotron d-flex flex-column justify-content-center p-5">
+        <h1 class="display-4 d-flex justify-content-center">Valentina Brambila Grando</h1>
+        <p class="lead d-flex justify-content-center">Georgia Institute of Technology | Computer Science '22</p>
+        <hr class="my-4">
+        <p class="d-flex justify-content-center">I am a coder and a dreamer. A bookworm and an artist. And I like to let my dreams spill into my keyboard.</p>
+    </div>
+    <div id="content" class="d-flex flex-column align-items-center p-3">
+      <div class="d-flex flex-column align-items-center contactinfo">
+          <h2 class="p-3">Contact Me</h2>
+          <div class="d-flex justify-content-around iconscontact p-3">
+            <a href="https://github.com/Valentinagrando" target="blank"><i class="fab fa-github fa-2x"></i></a>
+            <a href="https://www.linkedin.com/in/valentina-b-grando/" target="blank"><i class="fab fa-linkedin fa-2x"></i></a>
+            <a href="./resources/resume2.pdf". target="blank"><i class="far fa-file fa-2x"></i></a>          
```

Comment:
Maybe a fun challenge for later is creating a modal that shows your resume and a user can download it from there instead of just opening it in a new tab.

Reply:
No action on this just a over all idea